src/weather.js

```diff
@@ -0,0 +1,24 @@
+class Weather {
+    #weather;
+    constructor() {
+        this.#weather = "sunny";  
```

Comment:
I would change it to this.#weather = weather and add a setWeatherToSunny method to facilitate tests on other methods